### ISSUE TITLE
ENH: support int32 and float32

### ISSUE THREAD

**Issue by brentp** [soasme/nim-mustache#15]
Hi, I am attempting to add `seq[int32]` and `seq[float32]` to a context.
I have made this work by doing this:

```
import mustache
import mustachepkg/values

proc castValue*(values:seq[float32]): Value =
  var vs: seq[Value]
  for v in values:
    vs.add(v.float.castValue)
  Value(kind: vkSeq, vSeq: vs)
  
var ctx = newContext()
ctx["a"] = @[23'f32, 33'f32]
```
but it would be nice if this were supported in the library.
thanks for the nice package.

**Comment by soasme** [comment]
@brentp I have released this feature in nim-mustache v0.4.2. Please upgrade your project dependency.

Example:

```
# File: issue_15.nim

import mustache

let ctx = newContext()
ctx["f32seq"] = @[0'f32, 1'f32]

let s = """
{{# f32seq }}
{{ . }}
{{/ f32seq }}

{{ f32seq }}
"""

echo(s.render(ctx))
```

Output:

```
0.0
1.0

[0.0,1.0]
```

**Comment by brentp** [comment]
thank you! (and see my comment in the commit)
